docker-compose.yml

```diff
@@ -16,6 +16,9 @@ services:
       - 80:80 # Can be removed when running behind a web server or reverse proxy (like Apache, Nginx and else). See https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md
       - 8080:8080
       - 8443:8443 # Can be removed when running behind a web server or reverse proxy (like Apache, Nginx and else). See https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md
+    # network_mode: bridge # Is needed when you want to create the nextcloud-aio network usinh this file
```

Comment:
not sure if this line is needed, it works with it and I use it in most of my compose files

Reply:
I would not add this here

Reply:
do you mean below the envs?

Reply:
or do you mean the network_mode line?

Reply:
Network_mode should get removed, the rest I would indeed move below the envs

Reply:
done